packages/chakra-ui/src/AltDateWidget/AltDateWidget.tsx

```diff
@@ -113,7 +113,7 @@ function AltDateWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F exten
 
   return (
     <Box>
-      <Box display='flex' flexWrap='wrap' alignItems='center' justify='center'>
+      <Box display='flex' flexWrap='wrap' alignItems='center'>
```

Comment:
The addition of the `@chakra-ui/system` lockdown and/or the other bump caused this prop to be unnecessary. Removing it didn't seem to affect the UI at all